src/apps/bridge/app_main.cpp

```diff
@@ -335,6 +336,7 @@ static void defaultTask( void *parameters ) {
 
     debugSysInit();
     debugMemfaultInit(&usbCLI);
+    mcubootCliInit();
```

Comment:
Needed to access the `update sec` and `update clr` commands for testing testing image swapping on an isolated board. Will be removed before merge (unless we want to keep it)